share/ci/scripts/install_imath.sh

```diff
@@ -13,13 +13,11 @@ cd Imath
 
 git checkout ${TAG}
 
-mkdir build
-cd build
-cmake -DCMAKE_BUILD_TYPE=Release ..
-$SUDO cmake --build . \
+cmake -S . -B _build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF
+$SUDO cmake --build _build \
```

Comment:
This SUDO is curious, was it working around some sort of server-side issue?

Reply:
The sudo is necessary since the installation goes to /usr/local.

But to reduce similar confusion, I added comments to the 3 install scripts explaining their purpose. These install Imath, libdeflate, openjph as a part of the CI that validate that OpenEXR's cmake properly finds the official installation, or fetches/builds the dependency locally.